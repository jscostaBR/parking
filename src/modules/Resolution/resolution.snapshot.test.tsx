import * as React from 'react'
import { Resolutions } from '.'
import renderer from 'react-test-renderer'

describe('render Resolutions', () => {
  it('render correctly with type: registry', () => {
    const tree = renderer.create(
      <Resolutions type='registry' />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render correctly with type: payment', () => {
    const tree = renderer.create(
      <Resolutions type='payment' />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render correctly with type: getOut', () => {
    const tree = renderer.create(
      <Resolutions type='getOut' />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
