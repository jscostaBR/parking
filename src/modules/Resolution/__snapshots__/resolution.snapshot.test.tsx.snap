// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Resolutions render correctly with type: getOut 1`] = `
<div
  className="sc-bczRLJ djUyFE"
>
  <img />
  <span>
    Sa√≠da Liberada
  </span>
</div>
`;

exports[`render Resolutions render correctly with type: payment 1`] = `
<div
  className="sc-bczRLJ djUyFE"
>
  <img />
  <span>
    Pago!
  </span>
</div>
`;

exports[`render Resolutions render correctly with type: registry 1`] = `
<div
  className="sc-bczRLJ djUyFE"
>
  <img />
  <span>
    Registrado!
  </span>
</div>
`;
